gno.land/pkg/keyscli/root.go

```diff
@@ -34,6 +34,7 @@ func NewRootCmd(io commands.IO, base client.BaseOptions) *commands.Command {
 		client.NewExportCmd(cfg, io),
 		client.NewImportCmd(cfg, io),
 		client.NewListCmd(cfg, io),
+		client.NewUpdateCmd(cfg, io),
```

Comment:
Don't you think it's more appropriate after "delete"?

Reply:
I pushed a commit to put "update" after "delete". https://github.com/gnolang/gno/pull/2700/commits/0880d6929621484545b425d7bdeaa375924e7a3e